tests/e2e/performance/test_pvc_creation_deletion_performance.py

```diff
@@ -150,12 +193,22 @@ def test_pvc_creation_deletion_measurement_performance(
         )
         self.full_results.add_key("pvc_size", pvc_size)
         num_of_samples = 5
-        accepted_creation_time = 1
-        accepted_deletion_time = 2 if self.interface == constants.CEPHFILESYSTEM else 1
+        accepted_creation_time = (
+            3600 if self.interface == constants.CEPHBLOCKPOOL_THICK else 1
```

Comment:
A time out of 1 hour for 25Gi volume? Do you plan to redefine the timeout based on our test results?

Reply:
@keesturam  I've changed it to 600 secs ( upper limit of actial results)